src/sources/docker_logs.rs

```diff
@@ -863,10 +864,10 @@ impl ContainerLogInfo {
                     // Received log is not from before of creation
```

Comment:
Realize it isn't code you're modifying but if we do get to the bottom of when these conditions occur, it would be nice to improve the phrasing of this comment as well.

Reply:
I imagine this is just a sanity check to make sure that the log entry wasn't created before the container was. Like how could a container create a log entry if that container doesn't exist yet...